configs/setups/awiesm/awiesm.yaml

```diff
@@ -93,13 +93,19 @@ general:
         - '2.1-recom'
         - '2.1-paleodyn-2.5'
         - '2.6'
+        - '2.6-1'
```

Comment:
Same as above, why `-1` and not `.1`?

Reply:
afaik the extra . would not work, as it breaks some esm_master logic for what is a version. Did not test this recently though.

Reply:
For me, `.1` does not work at the moment. The the fesom config is taken from `fesom.yaml` and not `fesom-2.6.yaml`. I will have a closer look later today